.circleci/config.yml

```
@@ -198,6 +198,24 @@ commands:
             fi
             echo -n "$CIRCLE_SHA1" > commit_hash.txt
 
+  install_and_check_minimum_requirements:
+    parameters:
+      compiler:
+        description: "Compiler to check: gcc or clang"
+        type: string
+        default: ""
```

You could make it an enum:
```suggestion
        type: enum
        enum:
          - gcc
          - clang
```

Then the whole command reduces into:
```yaml
run: scripts/ci/install_and_check_minimum_requirements.sh "--<< parameters.compiler >>"
```

---

neat, thanks!